Scripts/Swift/SetUpTestFirebase.swift

```diff
@@ -0,0 +1,16 @@
+let ConstantTeamId = "<#teamId#>"
+let ConstantStagingFirebaseAppId = "<#stagingFirebaseAppId#>"
+let ConstantFirebaseTesterGroups = "<#group1#>, <#group2#>"
+let ConstantMatchRepo = "git@github.com:{organization}/{repo}.git"
```

Comment:
```suggestion
let teamId = "<#teamId#>"
let stagingFirebaseAppId = "<#stagingFirebaseAppId#>"
let firebaseTesterGroups = "<#group1#>, <#group2#>"
let matchRepo = "git@github.com:{organization}/{repo}.git"
```

Reply:
I will add placeholder to the end of these variables so they are more descriptive. `teamIdPlaceholder`.